hazelcast-jet-core/src/main/java/com/hazelcast/jet/Edge.java

```diff
@@ -335,11 +349,14 @@ public String toString() {
             case VARIABLE_UNICAST:
                 break;
             case PARTITIONED:
-                b.append(getPartitioner() instanceof Single ? ".allToOne()" : ".partitioned(...)");
+                b.append(getPartitioner() instanceof Single ? ".allToOne()" : ".partitioned(?)");
```

Comment:
I've given it a chance and `?` is just better, especially because the dot is also used with the usual Java meaning.